aip/cloud/2510.md

```diff
@@ -80,3 +84,4 @@ folders.
 ## Changelog
 
 - **2019-06-10**: Minor language and organization tweaks
```

Comment:
Could you flip the order of these before merging?

Reply:
Done